sdk/compiler/damlc/daml-opts/daml-opts-types/DA/Daml/Options/Types.hs

```diff
@@ -149,8 +151,8 @@ type ErrorOrWarning = Either TypeCheckerError.ErrorOrWarning LFConversion.ErrorO
 damlWarningFlagParser :: WarningFlags.DamlWarningFlagParser ErrorOrWarning
 damlWarningFlagParser =
   WarningFlags.combineParsers
-    TypeCheckerError.damlWarningFlagParser
-    LFConversion.damlWarningFlagParser
+    TypeCheckerError.damlWarningFlagParserTypeChecker
+    LFConversion.damlWarningFlagParserLFConversion
```

Comment:
Maybe don't need to qualify the import anymore, but fine to merge without

Reply:
It doesn't bother me much, let's keep it in